packages/plugins/model-train/cycle-gan-tensorflow-model-train/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@pipcook/plugins-cycle-gan-tensorflow-model-train",
```

Comment:
I don't like the name "cycle-gan-tensorflow-model-train", we need to figure out a more general one.